app/javascript/src/components/form/Derivatives.svelte

```diff
@@ -0,0 +1,84 @@
+<script>
+  import Tags from './Tags.svelte';
```

Comment:
I see the js-linter isn't work in Svelte files. It's fine for now, I will see if I can get it fixed.